components/event-sources/README.md

```diff
@@ -0,0 +1,55 @@
+# Knative event sources for Kyma
+
+Available event sources:
+
+* `HTTPSource`: for applications emitting HTTP events.
+
+## Running the controller inside a cluster
+
+```console
+$ ko apply -f config/
+```
+
+## Running the controller locally
+
+### Environment setup (performed once)
+
+Create the CustomResourceDefinitions for event source types.
+
+```console
+$ kubectl create -f config/ -l kyma-project.io/crd-install=true
+```
+
+Create the `kyma-system` system namespace. The controller watches ConfigMaps inside it.
+
+```console
+$ kubectl create -f config/100-namespace.yaml
+```
+
+Create ConfigMaps for logging and observability inside the system namespace (`kyma-system`)
+
+```console
+$ kubectl create -f config/400-config-logging.yaml
+$ kubectl create -f config/400-config-observability.yaml
+```
+
+### Controller startup
+
+Export the following mandatory environment variables:
+
+* **KUBECONFIG**: path to a local kubeconfig file (if different from the default OS location)
+* **SYSTEM_NAMESPACE**: set to "kyma-system" (see above)
```

Comment:
Do we provide sane defaults?

Reply:
I didn't set any default value because this requirement is deep inside Knative's sharedmain package, but it should be possible yes.

Reply:
i see, you already implemented it . Thx